packages/datagateway-common/src/homePage/homePage.component.tsx

```diff
@@ -0,0 +1,145 @@
+      color: '#FF6900',
+      fontWeight: 'bold',
+      paddingBottom: 10,
+    },
+    howItWorksGridItemImage: {
+      height: 200,
+      width: 200,
+      borderRadius: 200 / 2,
+      paddingBottom: 10,
+    },
+    howItWorksGridItemCaption: {
+      textAlign: 'center',
+      color: theme.palette.secondary.main,
+    },
+  });
+
+export type HomePageProps = WithStyles<typeof styles>;
+
+const HomePage = (props: HomePageProps): React.ReactElement => (
+  <div>
+    <div className={props.classes.bigImage}>
+      <img src={DatagatewayLogoWithText} alt={homePageContents['title']} />
+    </div>
+    <div className={props.classes.howItWorks}>
+      <Typography variant="h4" className={props.classes.howItWorksTitle}>
+        {homePageContents['how-label']}
+      </Typography>
+
+      <Grid container spacing={3}>
+        <Grid item sm={12} md={4} className={props.classes.howItWorksGridItem}>
+          <Typography
+            variant="h5"
+            className={props.classes.howItWorksGridItemTitle}
+          >
+            {homePageContents['explore-label']}
+          </Typography>
+          <img
+            src={ExploreImage}
+            alt=""
+            className={props.classes.howItWorksGridItemImage}
+          />
+          <Typography
+            variant="body1"
+            className={props.classes.howItWorksGridItemCaption}
+          >
+            {homePageContents['explore-description']}
+          </Typography>
+        </Grid>
+        <Grid item sm={12} md={4} className={props.classes.howItWorksGridItem}>
+          <Typography
+            variant="h5"
+            className={props.classes.howItWorksGridItemTitle}
+          >
+            {homePageContents['discover-label']}
+          </Typography>
+          <img
+            src={DiscoverImage}
+            alt=""
+            className={props.classes.howItWorksGridItemImage}
+          />
+          <Typography
+            variant="body1"
+            className={props.classes.howItWorksGridItemCaption}
+          >
+            {homePageContents['discover-description']}
+          </Typography>
+        </Grid>
+        <Grid item sm={12} md={4} className={props.classes.howItWorksGridItem}>
+          <Typography
+            variant="h5"
+            className={props.classes.howItWorksGridItemTitle}
+          >
+            {homePageContents['download-label']}
+          </Typography>
+          <img
+            src={DownloadImage}
+            alt=""
+            className={props.classes.howItWorksGridItemImage}
+          />
+          <Typography
+            variant="body1"
+            className={props.classes.howItWorksGridItemCaption}
+          >
+            {homePageContents['download-description']}
+          </Typography>
+        </Grid>
+      </Grid>
+    </div>
+  </div>
+);
+
+export const HomePageWithoutStyles = HomePage;
+export const HomePageWithStyles = withStyles(styles)(HomePage);
+
+export default HomePageWithStyles;
```

Comment:
If you make the changes that I suggested in the unit test file then that would mean that `HomePageWithoutStyles` will never be imported. I also don't think that `HomePageWithStyles` should be exported twice. You can refactor these four lines to the following:
```suggestion
export default withStyles(styles)(HomePage);
```